packages/gatsby-source-contentful/src/config.js

```diff
@@ -0,0 +1,10 @@
+export const restrictedNodeFields = [
+  // restrictedNodeFields from here https://www.gatsbyjs.org/docs/node-interface/
+  `id`,
+  `children`,
+  `parent`,
+  `fields`,
+  `internal`,
+  // Contentful Common resource attributes: https://www.contentful.com/developers/docs/references/content-delivery-api/#/introduction/common-resource-attributes
+  `sys`,
```

Comment:
`sys` is now restricted.

but we will reallow `contentful_id`. Others like `node_locale`, `updatedAt`, `createdAt`,...  never been in the list of restricted node names.